src/Organizer/OrganizerLDProjector.php

```diff
@@ -522,7 +522,10 @@ public function applyGeoCoordinatesUpdated(GeoCoordinatesUpdated $geoCoordinates
 
         $jsonLD = $document->getBody();
 
-        $jsonLD->geo = $geoCoordinatesUpdated->toJsonLd();
+        $jsonLD->geo = [
+            'latitude' => $geoCoordinatesUpdated->coordinates()->getLatitude()->toDouble(),
+            'longitude' => $geoCoordinatesUpdated->coordinates()->getLongitude()->toDouble(),
```

Comment:
Very small nit: I just noticed you named the method `coordinates` in a previous commit. Usually we prefix getters with `get`, so like `getCoordinates()`. It's not that I mind personally, it might just be a bit more consistent. But I'm approving the PR anyway because again, it's really minor and more of a style thing.